mautic/mautic
# Update plugin schema on plugin install

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/2938


[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When installing custom plugin which have entity does not create table as expected. This PR will resolve this issue. 

#### Steps to test this PR:
1. Try to install plugins with entity schema from plugins menu
2. Check the tables are created 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Try to install plugins with entity schema from plugins menu
2. Check the tables are created or not
